src/index.html

```diff
@@ -5,8 +5,528 @@
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--inactive"></div>
+            </div>
+            <span class="stars__reviews stars__reviews--possition">
+              Reviews: 5
+            </span> 
+          </div>
+
+          <div class="card__price-container">
+            <span class="card__price-text card__price-text--possition">
+              Price:
+            </span>
+            <span class="card__price-amount card__price-amount--possition">
+              $2,199
+            </span>
+          </div>
+
+          <div class="card__link">
+            <a
+              href="#"
+              alt="buy-link"
+              class="card__button card__button--possition"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="card__img-container">
+            <img 
+              class="card__image card__image--possition" 
+              src="images/imac.jpeg" 
+              alt="imac"
+            >
+          </div>
+
+          <div class="card__model-container">
+            <p class="card__model card__model--possition">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+          </div>
+
+          <div class="card__product-code-container">
+            <p class="card__product-code card__product-code--possition">
+              Product code: 195434
+            </p>
+          </div>
+
+          <div class="stars">
+            <div class="stars__block stars__block--possition">
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--inactive"></div>
+            </div>
+            <span class="stars__reviews stars__reviews--possition">
+              Reviews: 5
+            </span> 
+          </div>
+
+          <div class="card__price-container">
+            <span class="card__price-text card__price-text--possition">
+              Price:
+            </span>
+            <span class="card__price-amount card__price-amount--possition">
+              $2,199
+            </span>
+          </div>
+
+          <div class="card__link">
+            <a
+              href="#"
+              alt="buy-link"
+              class="card__button card__button--possition"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="card__img-container">
+            <img 
+              class="card__image card__image--possition" 
+              src="images/imac.jpeg" 
+              alt="imac"
+            >
+          </div>
+
+          <div class="card__model-container">
```

Comment:
This container is redudant

Reply:
> Check my comments

Hello, thanks for feedback
I answer below for each comment separately